mifosng-android/src/main/java/com/mifos/mifosxdroid/SettingsFragment.kt

```diff
@@ -54,6 +49,28 @@ class SettingsFragment : PreferenceFragment(), SharedPreferences.OnSharedPrefere
             true
         }
 
+        mInstanceUrlPref = findPreference(
+                getString(R.string.hint_instance_url)) as EditTextPreference
+        val instanceUrl = PrefManager.getInstanceUrl()
+        mInstanceUrlPref!!.text = instanceUrl
+        mInstanceUrlPref!!.isSelectable = true
+        mInstanceUrlPref!!.dialogTitle = "Edit Instance Url"
```

Comment:
It would be great if you could change this hardcoded string into string resources :)